lib/OfxParser/Parser.php

```diff
@@ -52,18 +50,17 @@ public function loadFromString($ofxContent)
     }
 
     /**
-     * Detect if the OFX file is on one line. If it is, add newlines automatically.
+     * Prepare OFX file contents.
```

Comment:
Can you put a better description here of what's going on to make it clearer please? Something like:

> Normalise newlines by removing and adding newlines only before opening tags